<launch>
    <arg name="pkg_name" value="pi_camera"/>
    <arg name="node_name" default="image_rect_full_ratio"/>

    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="local" default="false"
         doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>

    <group ns="$(arg veh)">
        <!-- run local -->
        <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen">
        </node>
    </group>
</launch>
